nsxt/resource_nsxt_policy_gateway_policy_test.go

```diff
@@ -386,7 +386,7 @@ func TestAccResourceNsxtGlobalPolicyGatewayPolicy_withSite(t *testing.T) {
 func TestAccResourceNsxtGlobalPolicyGatewayPolicy_withDomain(t *testing.T) {
 	name := "terraform-test"
 	siteName := getTestSiteName()
-	domainName := "new-domain"
+	domainName := fmt.Sprintf("new-domain-%d", getRandomSuffixInt())
```

Comment:
can we use `getAccTestResourceName()` for consistency with other tests?

Reply:
Sure, I thought that there's some reason that it wasn't used :)